content/testnet.md

```diff
@@ -0,0 +1,152 @@
+
+
+## Pre-requisites  
+
+### 1. Install Go 
+
+Any computer or personal laptop with 8GB of RAM and a decent internet connection can seamlessly run Rubix for token transfers / become validator / mining.
+
+- Download the latest Go binary package for your operating system from [Go's official site](https://golang.org/).  
+- Follow the installation instructions for your OS.  
+- Verify the installation and ensure the Go version is correct:
+
+   ```bash
+   go version
+   ```
+
+### 2. Download and Extract IPFS Kubo (v19)
+
+- Visit [IPFS Kubo releases](https://github.com/ipfs/kubo/releases) and download version 19 suitable for your operating system.
+- Locate the IPFS executable in the extracted files and verify if it has executable permissions.
+
+### 3. Clone the RubixGoPlatform Repository
+
+- Clone the repository:
+    ```bash
+    git clone https://github.com/rubixchain/rubixgoplatform.git
+    ```
+
+- Navigate to the repository directory
+    ```bash
+    cd rubixgoplatform
+    ```
+
+### 4. Build the RubixGoPlatform Executable
+- For Linux : 
+    ```bash
+    make compile-linux
+    ```
+
+- For Windows : 
+    ```bash
+    make compile-windows
+    ```
+
+- For MAC : 
+    ```bash
+    make compile-mac
+    ```
+
+Upon successful compilation, the rubixgoplatform executable will be created in a folder corresponding to your operating system. For example, on Linux, it will be located in the ./linux directory.
+
+### 5. Prepare the Necessary Files
+
+Ensure the following files are in the build directory:
+- rubixgoplatform executable
+- ipfs executable (from Kubo v19)
+- testswarm.key (provided by the Rubix team)
+
+
+## Joining the Testnet
+
+### Start a Rubix Node
+
+1. Open a terminal and navigate to the directory containing the `rubixgoplatform` executable.
+
+   ```bash
+   cd /path/to/rubixgoplatform
+   ```
+
+2. Start a Rubix node with the following command:
+   ```bash
+   ./rubixgoplatform run -s -testNet -p <node_name> -n <node_number> -grpcPort <grpc_port>
+    ```
+    
+    Replace the placeholders:
+
+    - \<node\_name\>: A unique name for your node.
+
+    - \<node\_number\>: Numeric identifier for the node.
+
+    - \<grpc\_port\>: Port number for gRPC communication.
+
+### Confirm the node is operational and connected to the testnet. Check for below in the logs.
+
+![Bootstrap connection check](/images/bootstrap_verify.png)
+
+
+### Create a DID (Decentralized Identifier) for the Node and register the DID
+
+1. Once the node is running, create a DID by executing:
+
+   ```bash
+   ./rubixgoplatform createdid -didType 4 -port <port_number>
+   ```
+
+2. Register the DID created by executing:
+   ```bash
+   ./rubixgoplatform registerdid -did <did_created> -port <port_number>
+    ```
+
+### Generate Test RBT Tokens
+To get test RBTs (Rubix Blockchain Test Tokens):
+ - Open the [faucet webpage](www.faucetlink.com) in browser
```

Comment:
provide faucet link

Reply:
The link is not complete, Thought would add it after completing.